packages/datagrid/src/components/DatasetSerializer/datasetSerializer.js

```diff
@@ -88,20 +103,28 @@ export function getQuality(qualityTotal, rowsTotal) {
  */
 export function getQualityValue(type) {
 	if (isArray(type)) {
-		return type.find(value => value[QUALITY_KEY] !== undefined)[QUALITY_KEY];
+		return get(type.find(value => value[QUALITY_KEY] !== undefined), QUALITY_KEY, null);
```

Comment:
I think you don't need to set `null` as default value.
(same for next line)

Reply:
And why do you use lodash `get()` ? The only reason I can see is that type can be null/undefined ?

Reply:
exact i have undefined i no get, it is suffisant. get avoid to do redundant check if object exist, take this property